backend/packages/Upgrade/src/api/controllers/ExcludeController.ts

```diff
@@ -194,33 +204,41 @@ export class ExcludeController {
    *           schema:
    *             type: object
    *             required:
-   *               - id
-   *               - type
+   *               - groups
    *             properties:
-   *               id:
-   *                type: string
-   *               type:
-   *                type: string
+   *              groups:
+   *                type: array
+   *                items:
+   *                  type: object
+   *                  properties:
+   *                    groupId:
+   *                      type: string
+   *                    type:
+   *                      type: string
    *       tags:
    *         - Exclude
    *       produces:
    *         - application/json
    *       responses:
    *          '200':
-   *            description: Exclude group from experiment
+   *            description: Exclude group globally
    *            schema:
-   *              $ref: '#/definitions/userExcludeResponse'
+   *              $ref: '#/definitions/groupExcludeResponse'
+   *          '401':
+   *            description: Authorization Required Error
+   *          '500':
+   *            description: Internal Server Error, Insert Error in database, JSON format is not valid
    */
-  @Put('/group')
-  public excludeGroup(@BodyParam('type') type: string, @BodyParam('id') id: string): Promise<ExplicitGroupExclusion> {
-    return this.exclude.excludeGroup(id, type);
+  @Post('/group')
+  public excludeGroup(@BodyParam('groups') groups: Array<{ groupId: string, type: string }>): Promise<ExplicitGroupExclusion[]> {
```

Comment:
Same here, we can use `excludeGroups`.

Reply:
The overall url which will be cerated after your change will be
`/explicitExclude/global/excludeGroups`
We already have `explicitExclude` in the path so `/group` seems fine to me.

Reply:
Yes, I am not saying that we change `/group` to `/groups` but we can change the function name from `excludeGroup` to `excludeGroups`